### DISCUSSION TITLE
Rendering issues in BoxPlotChartPlugin in preset-chart-xy

### DISCUSSION THREAD

**Discussion by vasan-vvs** [apache-superset/superset-ui#1158] [Questions]
I'm facing the below issue while trying new box plot plugin (https://github.com/apache-superset/superset-ui/tree/master/plugins/preset-chart-xy).
Can anyone please let me know if they have faced similar issue?
Superset version 1.0.1.

![Screen Shot 2021-06-08 at 4 16 27 PM](https://user-images.githubusercontent.com/18369896/121269646-0ffe8780-c875-11eb-8054-fc7f62c79288.png)

**Comment by villebro** [comment]
@vasan-vvs , this should now be fixed as of https://github.com/apache-superset/superset-ui/pull/1356
